jekyll/_cci2/reusing-config.md

```diff
@@ -972,6 +972,39 @@ condition | Y | Logic | [A logic statement](https://circleci.com/docs/2.0/config
 steps |	Y |	Sequence |	A list of steps to execute when the condition is falsy.
 {: class="table table-striped"}
 
+## Writing Inline Orbs
+
+When defining reusable configuration elements directly within your config, you can also wrap those elements within an inline orb. You may find inline orbs useful for development or for name-spacing elements that share names in a local config.
```

Comment:
Can I define reusable elements within my config, but NOT wrap them in an inline orb? 🔍 on that "can also wrap"

Reply:
I think my issue was coming here first from the orb page. When I look at the example in https://github.com/circleci/circleci-docs/pull/4506/files#diff-9811fd1b8b2059111e405f7877096579R77-R86 divided into separate yaml stanzas makes it looks like this command is a different file https://github.com/circleci/circleci-docs/pull/4506/files#diff-9811fd1b8b2059111e405f7877096579R77-R86